bokeh/sampledata/perceptions.py

```diff
@@ -4,7 +4,21 @@
 #
 # The full license is in the file LICENSE.txt, distributed with this software.
 #-----------------------------------------------------------------------------
-''' Provide provides access to probly.csv and numberly.csv
+''' Provide provides access to ``probly.csv`` and ``numberly.csv``.
```

Comment:
```suggestion
''' Provide access to ``probly.csv`` and ``numberly.csv``.
```

Reply:
This reminds me: In some instances the docstring begins with the verb "provide(s)", in most other instances it does not. Is there a reason for the difference? If not, I'd suggest going straigt with the description, whenever possible (such as "JSON data for co-occurrence of characters in Les Miserables").

Reply:
no reason except different people, different times. I agree that the direct description is best